config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
   resources :developers
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :developers
```

Comment:
You should specify methods.
Otherwise, all the routing are created including unnecessary ones.